src/bindgen/ir/ty.rs

```diff
@@ -592,50 +516,14 @@ impl Type {
         }
     }
 
-    fn nonzero_to_primitive(&self) -> Option<Self> {
-        let path = match *self {
-            Type::Path(ref p) => p,
-            _ => return None,
-        };
-
-        if !path.generics().is_empty() {
-            return None;
-        }
-
-        let name = path.name();
-        if !name.starts_with("NonZero") {
-            return None;
-        }
-
-        let (kind, signed) = match path.name() {
-            "NonZeroU8" => (IntKind::B8, false),
-            "NonZeroU16" => (IntKind::B16, false),
-            "NonZeroU32" => (IntKind::B32, false),
-            "NonZeroU64" => (IntKind::B64, false),
-            "NonZeroUSize" => (IntKind::Size, false),
-            "NonZeroI8" => (IntKind::B8, true),
-            "NonZeroI16" => (IntKind::B16, true),
-            "NonZeroI32" => (IntKind::B32, true),
-            "NonZeroI64" => (IntKind::B64, true),
-            "NonZeroISize" => (IntKind::Size, true),
-            _ => return None,
-        };
-
-        Some(Type::Primitive(PrimitiveType::Integer {
-            zeroable: false,
-            signed,
-            kind,
-        }))
-    }
-
     fn simplified_type(&self, config: &Config) -> Option<Self> {
         let path = match *self {
             Type::Path(ref p) => p,
             _ => return None,
         };
 
         if path.generics().is_empty() {
-            return self.nonzero_to_primitive();
+            return Some(self.clone());
```

Comment:
You can just return `None`, which is the shorthand for "no simplification needed", right?